src/JS/booking.js

```diff
@@ -18,11 +16,13 @@ const renderBookingSalonTitleDate = (salon, title, time) => {
         <h2>salong ${salon}</h2>
         <h1>${title}</h1>
         <h3>${time}</h3>
+        <div class="buy-button-container">
+            <button>Köp biljetter</button>
+        </div>
     `;
     
     bookingRenderContainer.appendChild(bookingInfoContainer);
```

Comment:
Another way to inject html with js is "insertAdjacentHTML". It's worth checking it out. It's really helpful :)

Reply:
I check that out. Thanx.